dist/obs-server.spec

```diff
@@ -750,9 +726,7 @@ usermod -a -G docker obsservicerun
 
 %package -n obs-container-registry
 Summary:        The Open Build Service -- container registry
-%if 0%{?suse_version} < 1210 && 0%{?suse_version:1}
```

Comment:
Is it fair to say that fedora builds of this spec file became uninteresting somewhen in 2013? Because parts of it pretend fedora support while others don't bother.